code/game/objects/structures/tables_racks.dm

```diff
@@ -124,7 +124,7 @@
 		if(!G.confirm())
 			return 0
 		G.affecting.loc = src.loc
-		G.affecting.Weaken(5)
+		G.affecting.Weaken(2)

```

Comment:
Cuffs are three, aren't they? I seem to remember that the detective's revolver was just enough to cuff someone with just the weaken, but only if you ninja'd them point-blank